docs/specs/layers.md

```diff
@@ -12,6 +12,11 @@ The `ty` property defines the specific layer type based on the following values:
 
 {schema_subtype_table:layers/all-layers:ty}
 
+The `st` property specifies a start time offset, while `sr` defines a time stretch factor,
+to be applied when evaluating animated properties pertaining to the layer:
+
+$t\prime = \dfrac{t}{stretch} - start$
+
```

Comment:
Currently sr only has an effect on precomp layers

Reply:
Yeah, that's unfortunate, how do you think is best to handle that?

1: Document the current semantics (precomp layers only), and move the property VisualLayer -> PrecompLayer.

This could be confusing, since the property is being exported by BM regardless of layer type, and also inconvenient: I imagine editors/tools may want to make adjustments, and having to update all keyframes or limit to precomps only seems awkward.

2. Document time stretch as applicable to all layers; also
  - update BM to export a value == 1 for regular layers (since the adjustment is already baked in)
  - update the players to process time stretch for all layer types
  - update the players to handle old/existing assets in some compatibility mode

I guess long term I would prefer we end up with option #2, for consistency.  But maybe that's for a later revision, so we don't introduce breaking changes in 1.0?

Reply:
Updated, now defined for precomp layers only.